bpf/flows.c

```diff
@@ -167,6 +169,7 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
             do_sampling = 0;
             return TC_ACT_OK;
         }
+        flow_sampling = filter_sampling;
```

Comment:
we probably can remove filter_sampling var
and just use
`flow_sampling = sampling;` like you did above

Reply:
Yeah, I was unsure, since `filter_sampling` is passed down to `check_and_do_flow_filtering` and the result is used to determine if a random sampling check is needed... I think it will work with a single var but didn't want to mess it up especially as there's no unit test

Reply:
done https://github.com/netobserv/netobserv-ebpf-agent/pull/691/commits/8c11876659a2e1a8137a27530c41b2d941384849